packages/app/src/app/components/Create/CreateBox/CreateBoxForm.tsx

```diff
@@ -38,6 +42,7 @@ export const CreateBoxForm: React.FC<CreateBoxFormProps> = ({
   const effects = useEffects();
   const nameInputRef = useRef<HTMLInputElement>(null);
   const { isPro } = useWorkspaceSubscription();
+  const isDraft = collectionId == null;
```

Comment:
```suggestion
  const isDraft = collectionId === undefined;
```

I'd keep consistency and stronger checks if this was defined as string | undefined in the top component